src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/indication/Indication.java

```diff
@@ -54,6 +55,12 @@
     private String pattern;
 
     /**
+     * @deprecated, kept for backwards compatibility.
+     */
+    @Deprecated
+    private transient FailureCause cause;
+
```

Comment:
Unfortunately, it's not possible. I tried - but Jenkins fails to deserialize `config.xml` in this case. I don't remember exactly the error, but it was something about "unknown field".